src/index.tsx

```diff
@@ -66,6 +67,14 @@ export interface ConfigureOptions {
   maxAge?: number;
 }
 
+export interface LogOptions {
+  tag: string | null;
+}
+
+function isLogOptions(obj: any): obj is LogOptions {
+  return (obj as LogOptions).tag !== undefined;
```

Comment:
Shouldn't it be something like `return obj && typeof obj === 'object' && 'tag' in obj;`